server/app/assets/javascripts/admin_predicate_configuration.ts

```diff
@@ -311,16 +311,16 @@ class AdminPredicateConfiguration {
         operatorOption.selected = false
       }
 
-      operatorOption.classList.toggle('hidden', shouldHide)
+      operatorOption.classList.toggle('hidden', !shouldHide)
```

Comment:
Why toggle versus removing it?  I'm not sure how toggling it (off<->on) makes sense here if we're intending to hide it.

Reply:
See docs on toggle: https://developer.mozilla.org/en-US/docs/Web/API/DOMTokenList/toggle#syntax

the second parameter controls whether it is added or removed.

Reply:
Oof this was another case of me not reading booleans correctly

Reply:
force is not the clearest param name for something like toggle.

LGTM